lib/stm32wba/BLE_TransparentMode/Core/Inc/app_conf.h

```diff
@@ -419,7 +347,7 @@ typedef enum
  *   - 2 : Debugger available in low power mode.
  *
  ******************************************************************************/
-#define CFG_DEBUGGER_LEVEL           (0)
```

Comment:
Is that expected to enable debugger by default ?
Is there an impact to be expected on power consumption?

Reply:
The goal of this PR is trying to have same code/features but with a new structure. 
And in this case we need to leave the same debugger conf (it means "0").
We could make same comments for other features (LED, BUTTONS, PM,...), but after this PR will be merged a new PR will be done to review systematically all the aspects of the integration

Reply:
So, it meas that users will not enjoy the same behavior once this PR is merged and until another one is done. Not sure it is acceptable

Reply:
This change should not introduce functional impact.

Reply:
I was no  clear in my argumentations, my mistake. With this PR we are not introducing any functional changement. Just rearranging the code according a proper directory structure using the Cube BLE Transparent mode application as a reference. As already done for STM32WB0. Enhancements will be introduced when available. This code rearrangement is mandatory to better highlight Cube FW code we are reusing in Zephyr. Hoping now it's enough clear.

Reply:
Just adding some configuration defines are inherited from Cube  (as CFG_LED_SUPPORTED or CFG_DEBUGGER_LEVEL) but are not used at all inside Zephyr integration. So their values have no impact at all on behavior.